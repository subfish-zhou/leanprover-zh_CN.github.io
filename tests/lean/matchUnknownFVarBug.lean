example (n? : Option Nat) : False := by
  match h:n?, h':n? with
  | some 0, _ => {}
