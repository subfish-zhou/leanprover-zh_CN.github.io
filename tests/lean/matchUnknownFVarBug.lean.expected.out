matchUnknownFVarBug.lean:2:2-2:7: error: missing cases:
(some (Nat.succ _)), Eq.refl, (some _), Eq.refl
none, Eq.refl, none, Eq.refl
matchUnknownFVarBug.lean:3:18-3:19: error: unsolved goals
n? : Option Nat
h : n? = some 0
x✝ : Option Nat
h' : some 0 = x✝
⊢ False
